packages/geocoder/src/geocoders/here.ts

```diff
@@ -0,0 +1,83 @@
+// Prettier does not support typescript annotation
+// eslint-disable-next-line prettier/prettier
+import type { Feature } from "geojson"
+import type { HereResponse, Item } from "../apis/here/types";
+import type { SingleGeocoderResponse, MultiGeocoderResponse } from "./abstract-geocoder";
+
+import Geocoder from "./abstract-geocoder";
+
+
+const hereResultTypeToPeliasLayerMap = {
+  place: "venue",
+  houseNumber: "address"
+};
+
+const convertHereToGeojson = (hereFeature: Item): Feature => {
+  const extraFields: { confidence?: number, addendum?: any } = {} ;
+  if (hereFeature.scoring) {
+    extraFields.confidence = hereFeature.scoring.queryScore;
+  }
+  if (hereFeature.categories) {
+    extraFields.addendum = {
+      categories: hereFeature.categories
+    }
+  }
+
+  return {
+    type: "Feature",
+    geometry: {
+      type: "Point",
+      coordinates: [hereFeature.position.lng, hereFeature.position.lat]
+    },
+    properties: {
+      source: "here",
+      label: hereFeature.address.label,
```

Comment:
(I'm tempted to create a var for `hereFeature.address`.)

Reply:
extracted a bunch of things, looks way better!